### PR TITLE
Update KISS logo, rates type warning and deadband management

### PR DESCRIPTION
**Pull request by fgiudice98**
Remove KISS logo placeholder text and update it with the official one

Thanks to Paul from flyduino.net

Changed the rates type change warning to add a simple cancel button and to show the warning always when changing rates type

### CODE REVIEW

File: src/js/tabs/pid_tuning.js [hunk 1]
```diff
@@ -2343,7 +2343,10 @@ TABS.pid_tuning.changeRatesType = function(rateTypeID) {
 
     if (self.currentRatesType !== RC_tuning.rates_type) {
         sameRatesType = false;
-        dialogRatesType.showModal();
+
+        if (!dialogRatesType.hasAttribute('open')) {
+            dialogRatesType.showModal();
+        }
```
Comment (mikeller) [inline src/js/tabs/pid_tuning.js]: I think the button assignments below should also be avoided if the dialog is already open.

Comment (fgiudice98) [reply src/js/tabs/pid_tuning.js]: Yep, you are right. Will fix now with a rebase and squash

File: src/js/tabs/pid_tuning.js [hunk 2]
```diff
@@ -2341,8 +2341,9 @@ TABS.pid_tuning.changeRatesType = function(rateTypeID) {
 
     self.currentRatesType = rateTypeID;
 
-    if (self.currentRatesType !== RC_tuning.rates_type) {
+    if ((self.currentRatesType !== RC_tuning.rates_type) && (!dialogRatesType.hasAttribute('open'))) {
```
Comment (mikeller) [inline src/js/tabs/pid_tuning.js]: The `(!dialogRatesType.hasAttribute('open'))` seems to be going too far now - it also covers `sameRatesType = false` and the `else` case.

**Comment by fgiudice98** [comment]
Latest commit if for changes in [betaflight pr #9579](https://github.com/betaflight/betaflight/pull/9579)

**Comment by Asizon** [comment]
This new change window system is really clear and concise, has all the options and is well explained. I just want to point one thing, before, it loaded the saved values of the saved type when you went back to it, now it doesn't load them, so you see default values of in the saved type and not the saved values. Perhaps it is better to continue as before in this aspect and load saved values of type if you go back to it, currently only load saved values using the button.

**Comment by fgiudice98** [comment]
New Actual rates logo thanks to @Asizon and @igormedrano
image

@mikeller what do you think about @Asizon idea to restore saved values when changing to the rates type saved in the model? I change to this way or just leave the button as the only way to return to the saved values?

**Comment by mikeller** [comment]
After looking at this, and thinking about it, I have come to the conclusion that we are overdoing the warning dialog here: Just the same as for everything else, if the user wants to restore the saved settings, they can change tabs and then change back, and there is no dedicated 'revert to saved settings' button needed:

image

This will also reduce the complexity of the code, and make it easier to maintain.

**Comment by Asizon** [comment]
Yeah I thought alotof about it and is a good option to simplify. Only change and cancel buttons with saved values availables when back to the saved type with selector

**Comment by fgiudice98** [comment]
Ok, will remove the "restore to saved values" button

**Comment by fgiudice98** [comment]
With the latest commit the "Return to saved values" is removed, leaving only "Change" and "Cancel" button. For reverting the saved values the "Refresh" button can be used (as in all the configurator tabs), or just rechange tab, or just unplug and replug the quad (:grin:)

**Comment by mikeller** [comment]
Looks good now, nice and simple!

**Pull request merged**